src/logtab.cpp

```diff
@@ -41,6 +41,16 @@ void LogTab::InitTreeView(const EventListPtr events)
 
     m_bar->ShowMessage(QString("%1 events loaded").arg(QString::number(m_treeModel->rowCount())), 3000);
 
+    // Display only time if all events occured on the same day
+    bool multipleDays = true;
+    if (events->size() >= 2) {
+       QModelIndex idx = ui->treeView->currentIndex();
+       auto firstDatetime = m_treeModel->index(0, COL::Time, idx.parent()).data(Qt::UserRole).toDateTime();
+       auto lastDatetime = m_treeModel->index(m_treeModel->rowCount()-1, COL::Time, idx.parent()).data(Qt::UserRole).toDateTime();
+       multipleDays = firstDatetime.date() != lastDatetime.date();
```

Comment:
Could also go with 24h diff but I’m not sure how often we look at logs around midnight for us to care ;)

Reply:
Yes, we could also hide the date if the whole log spans less than 24 hours. I was a bit conservative with this change and I am hiding the date only, if it does not add any additional information